utilities/bin/core/commands.dsc

```diff
@@ -0,0 +1,503 @@
+        - kill_queues
+        - killqs
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[null]>].if_null[true]>
+    script:
+        - if <proc[lib_has_permission].context[utilities.end_queues|<player.if_null[server_or_command_block]>].if_null[true]>:
+            - if <player.exists>:
+                - if <context.args.size> == 0:
+                    - define warn <script[lib_config].parsed_key[color.warning]>
+                    - define soft <script[lib_config].parsed_key[color.soft_server_notice]>
+                    - if <queue.list.size> > 1:
+                        - foreach <queue.list.exclude[<queue>]> as:queue:
+                            - narrate "<[warn]>Ending queue <[soft]><[queue].id>..."
+                            - queue stop <[queue]>
+                    - else:
+                        - narrate "<[warn]>No currently running queues!"
+                - else:
+                    - narrate <proc[lib_core_command_error].context[wrong_args|end_queues|0|<context.args.size>]>
+            - else:
+                - narrate <proc[lib_core_command_error].context[implicit]>
+        - else:
+            - narrate <proc[lib_core_command_error].context[permission]>
+#Simpler enchanting command
+denchant:
+    type: command
+    name: denchant
+    debug: false
+    description: A better enchantment command.<&nl>The denchant command will enchant your item with the enchantment specified, at the level you choose (or at level 1, if no level is specified.) Enchantments pull from properly registered enchantments, which are all vanilla enchantments, custom denizen enchantments, and enchantments from any plugin that registers correctly. Lore is also correctly added (or updated) on the item when required, and uses correct roman numeralization up to ~3000. The command will also let you know if the enchantment is out of range of vanilla, if the enchantment will function like expected on the item, or if the enchantment conflicts with an existing enchantment. The command will not halt if it encounters any of this, it just lets you know if you're trying to keep within vanilla enchantment bounds. The command also will not double enchant an item, and enchanting an item with a higher level or lower level enchant of the same enchant will raise/lower the enchant on the item appropriately. Lastly, when denchant is used on a regular book, it will create an enchantment book, rather than a regular book with an enchantment on it.
+    usage: <proc[lib_command_usage].context[lib_generic_data|command.usage.denchant]>
+    aliases:
+        - dchant
+    allowed help:
+        - determine <proc[lib_has_permission].context[utilities.denchant|<player.if_null[null]>].if_null[false]>
+    tab completions:
+        1: <server.enchantment_keys>
+        2: <proc[lib_numeric_list].context[<server.enchantment_start_level[<context.args.first>]>|<server.enchantment_max_level[<context.args.first>]>]>
```

Comment:
with the new enchantment update, the `server.enchantment_start_level` was deprecated in favor of [`enchantmenttag.min_cost[<level>]`](https://meta.denizenscript.com/Docs/Search/enchantmenttag.min_cost)